### ISSUE TITLE
Add missing error codes to `SheetMetalUnfolder.py`

### ISSUE THREAD

**Issue by luzpaz** [shaise/FreeCAD_SheetMetal#98]
There are certain places in the `SheetMetalUnfolder.py` that are missing error codes. 

https://github.com/shaise/FreeCAD_SheetMetal/blob/9bf7bac659b02aa9b5e920f74d70ac53e1d8cd01/SheetMetalUnfolder.py#L131-L133

https://github.com/shaise/FreeCAD_SheetMetal/blob/9bf7bac659b02aa9b5e920f74d70ac53e1d8cd01/SheetMetalUnfolder.py#L1859-L1860

https://github.com/shaise/FreeCAD_SheetMetal/blob/9bf7bac659b02aa9b5e920f74d70ac53e1d8cd01/SheetMetalUnfolder.py#L1510-L1513

https://github.com/shaise/FreeCAD_SheetMetal/blob/9bf7bac659b02aa9b5e920f74d70ac53e1d8cd01/SheetMetalUnfolder.py#L783-L785

**Comment by luzpaz** [comment]
Looks like @matthewdickinson's PR #10 was trying to address this

**Comment by luzpaz** [comment]
@shaise was this actually realized?

**Comment by shaise** [comment]
No, the PR was conflicting

**Comment by luzpaz** [comment]
Updated ticket with hyperlinked references in to the code

**Comment by alexneufeld** [comment]
@shaise This is probably not needed anymore, as we'll be using the new unfolder for the most part going forward

**Comment by shaise** [comment]
@luzpaz , Yes, with the new unfolder it is obsolete.  Thanks for helping me tidy things up!
